src/permissions/index.js

```diff
@@ -45,11 +47,19 @@ const permissions = shield(
     fallbackRule: allow,
 
     // Only allow the external errors when running in the local DEV environment
-    allowExternalErrors: process.env.IS_OFFLINE,
+    // OR when there is a CMR_ERROR a user needs to read
```

Comment:
It is important to be as detailed and precise as possible in comments. We are exposing CMR_ERRORs, but that logic is in the `fallbackError` function and this value has nothing to do with that.